NineChronicles.Headless/GraphTypes/StandaloneSubscription.cs

```diff
@@ -39,9 +39,16 @@ class PreloadStateType : ObjectGraphType<PreloadState>
         {
             private class PreloadStateExtra
             {
-                public string Type { get; set; }
-                public long CurrentCount { get; set; }
-                public long TotalCount { get; set; }
+                public string Type { get; }
+                public long CurrentCount { get; }
+                public long TotalCount { get; }
+
+                public PreloadStateExtra(string type, long currentCount, long totalCount)
```

Comment:
Why we need this constructor instead of previous property setters?

Reply:
The properties should be `non-nullable`, but, if we uses the property setter you said, it can't guarantee that it will be set always then it should be `nullable`. But the situtation seems not intended.

![image](https://user-images.githubusercontent.com/26626194/111720895-257d9a00-88a2-11eb-8439-93a0ffb891bb.png)